src/api/ratelimits.mdx

```diff
@@ -24,4 +24,8 @@ Every API request response includes the following headers:
 
 The rate limiter looks at the caller's identity instead of the bearer token or cookie. As such, the rate limit cannot be bypassed by generating multiple tokens.
 
-Rate limits for internal integrations may differ from other rate limits, and tend to be higher. We recommend using internal integration bearer tokens if you anticipate needing to make a large volume of calls.
+<Note>
+
+Polling the API for updates is a quick way to get rate limited, instead leverage our [webhooks](/product/integrations/integration-platform/webhooks/) if possible.
```

Comment:
"Leverage" is a word we try to avoid per our style guide and we want to avoid idioms. I've made some suggestions.

```suggestion
Polling the API for updates is likely to quickly trigger rate limiting. We recommend using our [webhooks](/product/integrations/integration-platform/webhooks/), if possible.
```